### ISSUE TITLE
Some image exercises?

### ISSUE THREAD

**Issue by nickeubank** [nickeubank/mids_coursera#76]
I don't want to hold up class two, but I feel like the addition of *some* exercises with images seem appropriate...

**Comment by nickeubank** [comment]
UGH I don't want to add work, but OF COURSE we need a James Webb image. And they can even be exposed to how we make false color images, since obvi it's all out of visible anyway. Get three wavelength slices of something cool. Walk them into making those RGB...

**Comment by nickeubank** [comment]
I was complaining about how I should get a RA not for any specific tasks but just for random data things in the common area, and @sarwaridas asked what I needed and then declared that she thought thus would be cool to do for fun so... She's on it! 

![duck](https://user-images.githubusercontent.com/9683693/201426284-4c12e0de-2445-4e5e-a6b5-4489de20fb79.gif)

**Comment by sarwaridas** [comment]
Yep should be fun!! Meet soon about what I need to do?

**Comment by nickeubank** [comment]
Nothing fancy—basically we have this [reading](https://github.com/nickeubank/mids_coursera/blob/main/class_2/week_3/25_images_as_matrices.ipynb), but no exercises, so I was thinking it'd be fun to get the actual JWST data (like, as numpy matrices) at a few different wavelengths. Just getting the data seems a bit annoying, so was looking for help. Then we can have the students plug in different wavelengths as RGB to see how false color images are created. But I know you have lots on your plate so don't worry too much about the exercise, it would just be great to have those matrices

**Comment by kylebradbury** [comment]
Two thoughts, but before any of those - thanks, Sarwari!

First, there are tutorials on how to make [pretty images](https://www.flickr.com/photos/geckzilla/) from the JWST such as [this tutorial](https://www.space.com/james-webb-space-telescope-image-editing) with a corresponding [video ](https://www.youtube.com/watch?v=8kGPGhbefhI) that shows how it's done with data from Hubble (the first 17 minutes are about the data gathering process. 

Second, it's possible this turns out to be a LOT... If so, another alternative is to reverse the direction of the camera and look at the earth. Sentinel 2 imagery from the European Space Agency is beautiful in the right regions and just by popping in different spectral bands into the RGB channels, you can make beautiful false-color imagery. In fact, there are some nice examples of band math that can be used (adding and dividing bands) that may use key skills the learners will be developing. Here are some examples of what the natural color and false color images can reveal from [Sentinel-hub playground](https://apps.sentinel-hub.com/sentinel-playground/?source=S2L2A&lat=26.616462035845263&lng=-80.42953491210938&zoom=11&preset=1_TRUE_COLOR&layers=B01,B02,B03&maxcc=0&gain=1.0&gamma=1.0&time=2021-09-01%7C2022-03-29&atmFilter=&showDates=false):
![sentinel-example](https://user-images.githubusercontent.com/4689093/201437525-dbfd46d1-dfde-41d0-ad50-ea7ef6532bec.gif)
(It even shows the band math on the left)

Just a few thoughts to share on this - let me know how I can help!

**Comment by sarwaridas** [comment]
Hi Nick, Kyle,

Thanks for sharing these sources! I looked over them briefly and it seems like the way to go would be to access MAST observations for the JWST mission -> download FITS data for a galaxy (for example) at different wavelengths --> process FITS images using software like FITS liberator to get them as a 2D array.  A few questions:
- What observation do you want me to look for? The article describes transforming an image of the Phantom Galaxy, and I can work with that for now.
- MAST has an option for minimum and maximum wavelength which I can go through to pick out images at different wavelengths. Is there a preference here too? Should I be looking for particular wavelengths?
- [Astropy](https://www.astropy.org/ ) seems to be able to import FITS images and convert them into 2D arrays so I could use that instead of FITS liberator. Are you familiar with this package or foresee any problems with it? For reference, I followed along a tutorial to explore Astropy/Astroquery [here](https://github.com/sarwaridas/JamesWebbSpaceTelescope/blob/main/Exploring_JWST_data.ipynb). I query through MAST and use the packages to get a matrix representation of the observation. It's hard to discern the observation so I clipped the data based on a threshold. Is this work along the lines of what you're looking for?

**Comment by nickeubank** [comment]
YES! That's perfect I think--a cool looking thing in the universe students will now feel they can interact with in a new way! 

(That's a super nova??)

Get a few slices if that, make rgb, and I think it's a perfect little exercise!

![image](https://user-images.githubusercontent.com/9683693/201449831-1deb1a09-f137-4240-9212-3a2a383aa2c9.jpeg)
